pkgs/development/python-modules/dep-logic/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, pdm-backend
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "dep-logic";
+  version = "0.0.4";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "pdm-project";
+    repo = "dep-logic";
+    rev = version;
+    hash = "sha256-AFiCNzHlo3BADqbjRBruA80cfM6Ytdb+gReEg5hUmro=";
+  };
+
+  nativeBuildInputs = [ pdm-backend ];
+
```

Comment:
```suggestion

propagatedBuildInputs = [
  packaging
];

```

Reply:
About the `packaging` dependency:
it seemed to build OK without it somehow and then I forgot to add it. Should just every package listed as a dependency always be included?

Reply:
> Should just every package listed as a dependency always be included?

Yes.